app/models/vita_partner.rb

```diff
@@ -45,6 +45,9 @@ class VitaPartner < ApplicationRecord
   scope :top_level, -> { where(parent_organization: nil).order(:display_name).order(:name) }
   scope :organizations, -> { where(parent_organization: nil) }
   scope :sites, -> { where.not(parent_organization: nil) }
+  has_many :child_sites, -> { order(:id) }, class_name: "VitaPartner", foreign_key: "parent_organization_id"
```

Comment:
is this called child_sites because we already have a scope called sites? I don't know if it would make it super confusing, but since the scope is a class method and the relationship is an instance method (I think), I'm pretty sure we could call both sites.

Reply:
That's right - called child_sites b/c the scope is called sites.

I thought about relying on the instance method vs. class method distinction. I think it's less confusing if they have separate names.